spectacles/cli.py

```diff
@@ -352,10 +360,18 @@ def sql(
     port,
     api_version,
     mode,
+    remote_reset,
 ) -> None:
     """Runs and validates the SQL for each selected LookML dimension."""
     runner = Runner(
-        base_url, project, branch, client_id, client_secret, port, api_version
+        base_url,
+        project,
+        branch,
+        client_id,
+        client_secret,
+        port,
+        api_version,
+        remote_reset,
```

Comment:
Since this is a SQL Validator-specific flag, instead of setting this in the constructor, this should be passed into `validate_sql` as an argument and used to instantiate the `SqlValidator`. Reason is we don't want to have to pass `remote_reset` for validators that are unrelated or don't care about this.

Reply:
We decided to leave this in the runner since it _does_ apply to all validators. Thanks @DylanBaker!